test/services/auth.test.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment node
+ */
+
+import { faker } from '@faker-js/faker';
+
+import { authenticateLogin } from '../../src/services/auth';
+import { user } from '../factories/user';
+import { createUser } from '../../src/app/actions/user';
+
+describe('Auth:', () => {
+  describe('given correct credentials', () => {
+    it('returns user', async () => {
+      const userPayload = { name: user.name, email: user.email, password: user.password };
```

Comment:
Add a line break between each block to delimit each part of [the AAA structure of tests](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80).

```suggestion
      const userPayload = { name: user.name, email: user.email, password: user.password };
      
```